src/client/driver/status.js

```diff
@@ -4,6 +4,7 @@ export default class DriverStatus extends Assignable {
     constructor (obj) {
         super(obj);
 
+        this.id              = Date.now();
```

Comment:
We already use `Math.random` in various places, I don't think someone will replace it. However it's possible that people will replace `performance.now`